tests/init_config_test.rs

```diff
@@ -0,0 +1,57 @@
+use scroll_proving_sdk::{config::Config, prover::CircuitType};
+use sindri_scroll_sdk::prover::CloudProver;
+use std::env::set_var;
+
+// Ensures that configuration file loading does not require environment variables
+#[tokio::test]
+async fn test_config_without_envs() {
+    let default_config_path = "tests/test_data/default_config.json";
+    let cfg: Config = Config::from_file_and_env(default_config_path.to_string())
+        .expect("Issue loading test configuration file");
+
+    // Check a few values for consistency with test_data file
+    assert_eq!(cfg.prover_name_prefix, "sindri_");
+    assert_eq!(cfg.prover.circuit_type, CircuitType::Bundle);
+}
+
+// Ensures that various environment variable overrides are successful
+#[test]
+fn test_config_with_envs() {
+    // Establish some environment variable overrides
+
+    set_var("N_WORKERS", "10");
+    set_var("COORDINATOR_BASE_URL", "my-special-coordinator");
```

Comment:
The string `"my-special-coordinator"` is used twice in this test. Normalizing this as a single constant variable would be good. Frankly, same idea for the `10_usize` and `"10"` pair of values.

Reply:
nice